spec/controllers/finders_controller_spec.rb

```diff
@@ -425,7 +425,7 @@
 
   describe "with legacy query parameters for publications" do
     before do
-      search_api_request
+      search_api_request(search_api_app: "search-api-v2")
```

Comment:
I think the tests are failing because the stub for v1 `search_api_request` hasn't been added back.
I'd leave both in for the break glass to make switching easier